### PR TITLE
Remove unused files, configure conditional routes for services, some quick fixes

### PR DESCRIPTION
**Pull request by anakin001**
Screenshot from 2023-07-24 15-22-00

### CODE REVIEW

**Comment by anakin001** [comment]
image

**Comment by anakin001** [comment]
image

File: src/util/buildTransactions.ts [hunk 1]
```diff
@@ -174,7 +174,7 @@ export class BuildTransactions {
 
     aggregateComplete(innerTxn: InnerTransaction[]): AggregateTransaction {
 
-        return this.transactionBuilderFactory.aggregateCompleteV1()
+        return this.transactionBuilderFactory.aggregateCompleteV1 ()
```
Comment (sleepyOwl14) [inline src/util/buildTransactions.ts:177]: no extra spacing before function invoke,
if it is eslint, please set it to off in .eslintrc file
```
"rules": {
  "func-call-spacing": "off"
}
```

**Review by sleepyOwl14 — CHANGES_REQUESTED** [review]

File: src/util/buildTransactions.ts (hunk 1 above)
Comment (anakin001) [reply src/util/buildTransactions.ts]: resolved

**Review by sleepyOwl14 — APPROVED** [review]

**Pull request merged**